Dangerfile.swift

```diff
@@ -43,21 +47,23 @@ if testFiles.isEmpty {
 
 modified.forEach {
     warn("\($0)")
-    warn("\($0.fileType!)")
-    warn("\($0.fileType!.rawValue)")
-    warn("\($0.fileType!.extension)")
 
     if let fileTypeModified = $0.fileType, fileTypeModified.extension == "podspec" {
         warn("#{danger.git.html_link('\($0)'')}")
     }
 
     if $0 == "Gemfile" || $0 == "Gemfile.lock" {
-        warn("#{danger.git.html_link('Gemfile.lock')} ")
+        warn("#{danger.github.html_link(\($0))} was edited.")
     }
 
     if $0 == "Podfile" || $0 == "Podfile.lock" {
         warn("#{danger.git.html_link('Podfile')} was edited but #{danger.git.html_link('Podfile.lock')} wasn't. commit the #{danger.git.html_link('Podfile.lock')} changes.")
     }
 }
 
-message("🎉 The PR added \(additions) and removed \(deletions) lines. 🗂 \(changedFiles) files changed.")
\ No newline at end of file
+message("🎉 The PR added \(additions) and removed \(deletions) lines. 🗂 \(changedFiles) files changed.")
+
+slather.configure("/Users/douglas.garcia/Documents/GitHub/danger-bot/TravisBot/TravisBot.xcodeproj", "TravisBotTests")
+slather.notify_if_coverage_is_less_than(minimum_coverage: 80)
+slather.notify_if_modified_file_is_less_than(minimum_coverage: 60)
+slather.show_coverage
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/dbgarcia/danger-bot/blob/4ad3f65193926f1211a9cfb888b46cbbba684b48/Dangerfile.swift#L69"></span>Files should have a single trailing newline.<br />
<code>trailing_newline</code> <code>Dangerfile.swift:69</code></del></td>
    </tr>
  </tbody>
</table>